lib/ts/querier.ts

```diff
@@ -182,14 +192,18 @@ export default class Querier {
             throw result;
         }
 
-        return postAPIHook === undefined
-            ? result
-            : await postAPIHook({
-                  requestInit,
-                  url: modifiedUrl,
-                  fetchResponse: result,
-                  userContext,
-              });
+        const reponseForPostAPI = result.clone();
```

Comment:
Maybe you want to do this clone inside the below `if` statement?

Reply:
Done